Extension/README.md

```diff
@@ -56,6 +56,7 @@ The extension has platform-specific binary dependencies, therefore installation
 `cpptools-linux-aarch64.vsix` | Linux ARM 64-bit
 `cpptools-osx.vsix` | macOS
 `cpptools-win32.vsix` | Windows 64-bit & 32-bit
+`cpptools-win-arm64.vsix` | Windows ARM64
```

Comment:
Also update the other READMD.md in the root folder

Reply:
Good catch.